angr/analyses/cfg/cfg_fast.py

```diff
@@ -1927,7 +1943,7 @@ def _create_jobs(self, target, jumpkind, current_function_addr, irsb, addr, cfg_
 
         if type(target) is pyvex.IRExpr.Const:  # pylint: disable=unidiomatic-typecheck
             target_addr = target.con.value
-        elif type(target) in (pyvex.IRConst.U32, pyvex.IRConst.U64):  # pylint: disable=unidiomatic-typecheck
+        elif type(target) in (pyvex.IRConst.U8, pyvex.IRConst.U16, pyvex.IRConst.U32, pyvex.IRConst.U64):  # pylint: disable=unidiomatic-typecheck
```

Comment:
Why `U8`?

Reply:
Some day I will finish AVR support? :D

Reply:
...and technically this particular line should be an isinstance, since we can have arbitrary width things in gymrat, but... yeah i'm leaving this as it is for now.